aiidalab_widgets_base/computational_resources.py

```diff
@@ -902,7 +913,14 @@ def on_setup_computer(self, _=None):
         )
         try:
             computer = computer_builder.new()
-            self._configure_computer(computer)
+            if self.transport.value == "core.ssh":
+                self._configure_computer_ssh(computer)
+            if self.transport.value == "core.local":
+                self._configure_computer_local(computer)
+            else:
+                self.message = (
+                    f"WARNING: Transport value {self.transport.value} not recognized!"
```

Comment:
I think it would be safer to throw an error here. I guess a computer that is not configured is not usable?

Reply:
> I guess a computer that is not configured is not usable?

One can configure it later, of course. But that is not the purpose of the widget. The widget should create a computer that is ready to be used.

> I think it would be safer to throw an error here.

I agree. I made a suggestion below.

Reply:
Agreed, donw